util/topgen/intermodule.py

```diff
@@ -51,7 +52,7 @@ def elab_intermodule(topcfg):
     list_of_intersignals = []
 
     if "inter_signal" not in topcfg:
-        topcfg["inter_signal"] = {}
+        topcfg["inter_signal"] = OrderedDict({})
```

Comment:
You can remove the curly braces here and in similar cases in this patch.

Reply:
Thanks! revised.